yb-voyager/src/srcdb/source.go

```diff
@@ -289,3 +294,53 @@ func (source *Source) extrapolateDSNfromSSLParams(DSN string) string {
 
 	return DSN
 }
+
+func (source *Source) PrepareSSLParamsForDebezium(exportDir string) error {
+	switch source.DBType {
+	case "postgresql":
+		if source.SSLKey != "" {
+			targetSslKeyPath := filepath.Join(exportDir, "metainfo", "ssl", "key.der")
+			err := dbzm.WritePKCS8PrivateKeyPEMasDER(source.SSLKey, targetSslKeyPath)
```

Comment:
Same doubts as earlier, what if the key is already provided in DER format?

Reply:
replied above